parm/config/config.defaults.s2sw

```diff
@@ -15,21 +14,27 @@ FHOUT_HF_GFS=-1
 min_seaice="1.0e-6"
 use_cice_alb=".true."
 
-# config.fv3
-DELTIM=300
-layout_x_gfs=8
-layout_y_gfs=8
-WRITE_GROUP_GFS=1
-WRTTASK_PER_GROUP_GFS=24
-#The settings below will result in S2SWA running 35 days under 8 hours wallclock on hera
-#layout_x_gfs=24
-#layout_y_gfs=16
-#WRTTASK_PER_GROUP_GFS=86
-WRTIOBUF="32M"
-MEDPETS=300
-
+# config.fv3  # TODO: This is hard-wired for P8 and needs to be refactored.  For now, use case C384
+case "${CASE}" in
+  "C384")
+    DELTIM=300
+    layout_x_gfs=8
+    layout_y_gfs=8
+    WRITE_GROUP_GFS=1
+    WRTTASK_PER_GROUP_GFS=24
+    #The settings below will result in S2SWA running 35 days under 8 hours wallclock on hera
+    #layout_x_gfs=24
+    #layout_y_gfs=16
+    #WRTTASK_PER_GROUP_GFS=86
+    WRTIOBUF="32M"  # TODO: This value is for P8 w/ C384.  Why not update/set this as default in config.fv3 C384?
+    MEDPETS=300  # TODO: P8 wants to use 300 instead of ATMPETS = layout_x * layout_y * 6
```

Comment:
FYI: The reason we use less than ATMPETS is it's been found to be more efficient, so this is likely not going to be limited to a "p8" case.   That'll be for future GEFS and GFS once optimized settings are determined again (the current ones here were "sufficient" but not "optimal").